docs/index.rst

```diff
@@ -4,9 +4,10 @@
 build
 *****
 
-A simple, correct :pep:`517` build frontend.
+A simple, correct Python build frontend.
 
-build will invoke the :pep:`517` hooks to build a distribution package.
+build implements handling for pyproject.toml-based builds, invoking
```

Comment:
I think this is a bit convoluted, maybe:

```suggestion
build manages pyproject.toml-based builds, invoking
```

Reply:
TBH, I mostly filed this as it seemed like an easy change wherein ~anything is better than status quo (which is plain wrong).

If we're word-smithing, I'd prefer that someone else drive this one over the line.

Reply:
We can merge this as-is and somebody else can copy-edit it in a subsequent PR if you prefer.

Reply:
That would work for me, or I'm happy for maintainers to push copy edits here directly. :)

Reply:
We can do that, thank you for the initial contribution!